protocols/authentication.asciidoc

```diff
@@ -0,0 +1,75 @@
+[[Authentication]]
+=== Authentication
+
+PowerTrade requires clients to use API keys in order to access the APIs.
+
+There are two components to an API key: the key itself and an API `secret_key`. Both are needed in order to authenticate and use API services.
+
+The `secret_key` is used to sign session protocol messages sent to the platform’s backend during <<login>>.
+
+The details about signing procedure can be found below.
+
+==== Obtaining API keys and secrets
+
+// TODO - add step by step instructions
+API keys can be obtained here <link>.
+
+==== Generating credentials_secret
+
+To initiate a session after establishing a WSS connection to a service, users must send a <<login>> message to the backend using their email as the `credentials` and a generated `credentials_secret`.
```

Comment:
@MattNguyen : I found the following style is more suitable, please consider it too:
```
<<login>> => <<login, login>>
```